packages/react-components/review/react-components.api.md

```diff
@@ -322,19 +322,19 @@ export interface GridLayoutProps {
 // @public
 export type GridLayoutType = 'standard';
 
-// @beta
-export const IdentifierProvider: (props: IdentifierProviderProps) => JSX.Element;
+// @internal
+export const _IdentifierProvider: (props: _IdentifierProviderProps) => JSX.Element;
```

Comment:
Are we planning to release this as part of the API? (If so would `@beta` be better than `@internal`?)

Reply:
This one really is @internal -- to be only used for automation tests, and we don't give API guarantees.

@beta is intended for API that will eventually become @public